### ISSUE TITLE
Parsing error in get_rtt_loss

### ISSUE THREAD

**Issue by IgnusG** [Chadsr/NordVPN-NetworkManager#102]
Got this error last time running a sync update:

```
"""
Traceback (most recent call last):
  File "/usr/lib/python3.6/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/usr/lib/python3.6/site-packages/nordnm/benchmarking.py", line 43, in compare_server
    score, load, latency = get_server_score(server, ping_attempts)
  File "/usr/lib/python3.6/site-packages/nordnm/benchmarking.py", line 26, in get_server_score
    rtt, loss = utils.get_rtt_loss(ip_addr, ping_attempts)
  File "/usr/lib/python3.6/site-packages/nordnm/utils.py", line 108, in get_rtt_loss
    loss = int(split_info[5].split('%')[0])
ValueError: invalid literal for int() with base 10: '33.3333'
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/bin/nordnm", line 11, in <module>
    load_entry_point('nordnm==0.5.4', 'console_scripts', 'nordnm')()
  File "/usr/lib/python3.6/site-packages/nordnm/__main__.py", line 26, in main
    nordnm.NordNM()
  File "/usr/lib/python3.6/site-packages/nordnm/nordnm.py", line 207, in __init__
    self.sync(args.update_configs, args.preserve_vpn, args.slow_mode)
  File "/usr/lib/python3.6/site-packages/nordnm/nordnm.py", line 378, in sync
    if self.sync_servers(preserve_vpn, slow_mode):
  File "/usr/lib/python3.6/site-packages/nordnm/nordnm.py", line 581, in sync_servers
    best_servers, num_success = benchmarking.get_best_servers(valid_server_list, ping_attempts, valid_protocols, valid_categories, slow_mode)
  File "/usr/lib/python3.6/site-packages/nordnm/benchmarking.py", line 100, in get_best_servers
    for i, result in enumerate(pool.imap(partial(compare_server, best_servers=best_servers, ping_attempts=ping_attempts, valid_protocols=valid_protocols, valid_categories=valid_categories), server_list)):
  File "/usr/lib/python3.6/multiprocessing/pool.py", line 735, in next
    raise value
ValueError: invalid literal for int() with base 10: '33.3333'
```

**Comment by Chadsr** [comment]
Thanks for the report.

I have switched to casting loss as a float. Not sure why I had casted it to an int previously, that was a little naive of me.

This problem should therefore be gone in the next patch update shortly to be released (0.5.5).

**Comment by IgnusG** [comment]
Thanks for the fix!
